LICENSE

```diff
@@ -45,7 +45,7 @@ modification, are permitted provided that the following conditions are met:
   b. Redistributions in binary form must reproduce the above copyright
      notice, this list of conditions and the following disclaimer in the
      documentation and/or other materials provided with the distribution.
-  c. Neither the name of Diofant, or SymPy nor the names of its contributors
```

Comment:
Niether the name Diofant, SymPy nor...

But isn't this the license that has to be included from Diofant verbatim? If so, that is not a SymPy edit to make.